### PR TITLE
Change method for downloading users from post to get

### PR DESCRIPTION
**Pull request by carolinegreen**
Allow list of interested users to be downloaded.  Changed method used in earlier pull request by @pcraig3 https://github.com/alphagov/digitalmarketplace-admin-frontend/pull/162 from post to get.

### CODE REVIEW

File: app/templates/download.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{% from 'macros/page_heading.html' import page_heading %}
```
Comment (robyoung) [inline app/templates/download.html]: Why do we need a separate page for this? Could we just link directly to the downloads on when we list the frameworks?

**Comment by carolinegreen** [comment]
screen shot 2016-01-06 at 10 04 35

Using list of links instead of radio buttons as this isn't a form.

**Comment by carolinegreen** [comment]
screen shot 2016-01-06 at 10 06 29

List of users can be downloaded on each page corresponding to each framework.

File: app/templates/download.html (hunk 1 above)
Comment (carolinegreen) [inline app/templates/download.html]: @robyoung We could, I assumed there was some reasoning behind downloading being a separate action.  If there isn't I'm happy to change it though.

Comment (quis) [inline app/templates/download.html]: I vote for one page. And use [browse list](http://alphagov.github.io/digitalmarketplace-frontend-toolkit/browse-list.html) for the list of frameworks.

Comment (carolinegreen) [inline app/templates/download.html]: @quis Didn't know about browse list, will use that.

**Comment by carolinegreen** [comment]
screen shot 2016-01-06 at 14 00 08

Now using browse-list to show downloadable list of users on frameworks.

File: tests/app/main/views/test_users.py [hunk 2]
```diff
@@ -169,3 +175,117 @@ def test_should_show_deactivate_button(self, data_api_client):
...
+
+    _valid_framework = {
+        'name': 'G-Cloud 7',
+        'slug': 'g-cloud-7',
+        'status': 'live'
+    }
+
+    _invalid_framework = {
+        'name': 'G-Cloud 8',
+        'slug': 'g-cloud-8',
```
Comment (TheDoubleK) [inline tests/app/main/views/test_users.py]: By August this will be a valid framework :)

**Comment by TheDoubleK** [comment]
Tested locally and it looks good to me.

It would be nice to try to get this deployed through to prod this afternoon as a new email list needs to be generated right after DOS clarification questions close (see https://www.pivotaltracker.com/story/show/110969430) - it would be nice if a non-dev could perform this chore!

**Comment by TheDoubleK** [comment]
As a sanity check though it would definitely be worth generating an email list using the existing script and from this new admin page, and checking that the lists tally up.

File: app/templates/download_users.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        items = [
+          {
+              "title": framework.label,
+              "link": url_for('.download_users', framework_slug=framework.value)
+          }
+        ]
+      %}
+        {% include "toolkit/browse-list.html" %}
+      {% endwith %}
+        <!--<li><a href="{{  url_for('.download_users', framework_slug=framework.value) }}">{{ framework.label }}</a></li>-->
```
Comment (TheDoubleK) [inline app/templates/download_users.html]: Could delete this commented out code.

**Comment by TheDoubleK** [comment]
:+1: from me, but might be worth checking that @quis and @robyoung are happy now.

**Comment by TheDoubleK** [comment]
Should @pcraig3's earlier PR be closed now too?

**Comment by carolinegreen** [comment]
@TheDoubleK Probably, will close it.

**Comment by quis** [comment]
Latest screenshot looks good to me.

File: app/templates/download_users.html (hunk 3 above)
Comment (carolinegreen) [inline app/templates/download_users.html]: Thought I had! :o It's gone now.

**Pull request merged**